### DISCUSSION TITLE
Release v1.14.5 / 5.69.5

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#4107] [Announcements]
This build fixes various bugs and adds some minor improvements.

For a full list of changes and fixes please review the [full change log](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#1145--5695---2024-07-23).

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.14.5'>Release v1.14.5 / 5.69.5</a>.</em>

**Comment by qk1ka** [comment]
https://forum.xanasoft.com/threads/bug-steam-and-game.842/post-2051 - look at this
